securedrop/source_templates/index.html

```diff
@@ -88,15 +88,14 @@ <h2 class="welcome-text">
     {# Warning bubble to help TB users disable JavaScript with NoScript.
     Included here so the images can preload while the user is first
     reading the page. Hidden by default. #}
-    <div class="bubble" id="security-slider-info" class="info">
-        <div class="row-reverse"> <img id="security-slider-info-close" class="close" src="{{ url_for('static', filename='i/font-awesome/black/times.svg') }}" width="20px" height="20px"></div>
-      <p>{{ gettext('You appear to be using the Tor Browser. You can move the Security Slider to Safest in four steps.') }}</p>
+    <div class="bubble" id="security-setting-info" class="info">
       <ol>
-        <li>{{ gettext('Click the <img src="{icon}" alt="shield icon"> shield icon in the toolbar above').format(icon=url_for("static", filename="i/shield.png"))  }}</li>
-        <li>{{ gettext('Click <strong>Advanced Security Settings...</strong>') }}</li>
-        <li>{{ gettext('Click <strong>Safest</strong>') }}</li>
-        <li>{{ gettext('<a href="/">Click here</a> to refresh the page') }}</li>
+        <li>{{ gettext('Click the <img src="{icon}" alt="shield icon"> in the toolbar above').format(icon=url_for("static", filename="i/font-awesome/black/guard.svg"))  }}</li>
```

Comment:
Shield icon is incorrect, should be outlined shield, not solid. It changes to solid once `Safest` is selected.

Reply:
@zenmonkeykstop We're advising the user to perform the action when the browser is _not_ in `safest`, though—does the whole banner not go away once the browser is at `safest`? It should.

Reply:
Yes, that's what I'm getting at. When the browser is in `Standard` mode, the shield icon in the toolbar is just an outline. The instructions in the popup ask the user to click on the solid shield icon, which isn't displayed at that point.

Reply:
Thank you for the clarification and for catching that! Ya, I couldn't find any specs (or even the artwork itself) for its implementation anywhere, so just kinda guessed. Submitted a new comment in the OG issue #4465 with updated SVG code.